doc/user_guide/search_paths.md

```diff
@@ -0,0 +1,103 @@
+Search paths
+============
+It is possible to extend DLite with new datamodels, code generation templates and plugins, by appending to corresponding search path.  The table below lists the different types of search paths that are available in DLite.
+
+| Search path type        | Description                                           |
+|-------------------------|-------------------------------------------------------|
+| storages                | Storage URLs or directory paths to datamodels         |
+| templates               | Directory paths to code generator templates           |
+| storage_plugins         | Directory paths to storage plugins written in C       |
+| mapping-plugins         | Directory paths to mapping plugins written in C       |
+| python-storage-plugins  | Directory paths to storage plugins written in Python  |
+| python-mapping-plugins  | Directory paths to mapping plugins written in Python  |
+| python-protocol-plugins | Directory paths to protocol plugins written in Python |
```

Comment:
```suggestion
| Search path type        | Description                                           |
|-------------------------|-------------------------------------------------------|
| storages                | Storage URLs or directory paths to datamodels         |
| templates               | Directory paths to code generator templates           |
| storage_plugins         | Directory paths to storage plugins (drivers) written in C       |
| mapping-plugins         | Directory paths to mapping plugins written in C       |
| python-storage-plugins  | Directory paths to storage plugins (drivers) written in Python  |
| python-mapping-plugins  | Directory paths to mapping plugins written in Python  |
| python-protocol-plugins | Directory paths to protocol plugins written in Python |
```

Reply:
Sorry about the alignement at the right. It is better to do this in a proper editor. Here I can jsut guess,